docs/develop/asynchronous-programming-in-office-add-ins.md

```diff
@@ -394,5 +394,5 @@ In both optional parameter examples, the  _callback_ parameter is specified as t
 ## See also
 
 - [Understanding the JavaScript API for Office](understanding-the-javascript-api-for-office.md) 
-- [JavaScript API for Office](https://dev.office.com/reference/add-ins/javascript-api-for-office)
+- [JavaScript API for Office](https://docs.microsoft.com/javascript/api/overview/office)
```

Comment:
I think this should point to: https://docs.microsoft.com/javascript/office/javascript-api-for-office